apps/emqx_bridge/src/schema/emqx_bridge_schema.erl

```diff
@@ -212,3 +213,26 @@ status() ->
 
 node_name() ->
     {"node", mk(binary(), #{desc => ?DESC("desc_node_name"), example => "emqx@127.0.0.1"})}.
+
+webhook_bridge_validator(
```

Comment:
Could it be a bit unintuitive to have two different settings that should be aways set equal?
M.b. one setting should suppress the other one and emit a warning?

Reply:
Another idea I had was to use the converter to copy the greater value to both fields automatically and log that change.

Reply:
Changed to this second approach; now the bridge won't fail, just use the greatest of the two possible values.